lib/galaxy/managers/users.py

```diff
@@ -608,8 +608,7 @@ def send_reset_email(self, trans, payload, **kwd):
                 except Exception as e:
                     log.debug(body)
                     return f"Failed to submit email. Please contact the administrator: {util.unicodify(e)}"
-            else:
-                return "Failed to produce password reset token. User not found."
+        return None
```

Comment:
```suggestion
        if not reset_user:
            log.warning(f"Failed to produce password reset token. User with email '{email}' not found.")
        return None
```
In case there is some debugging to be done.